README.md

```diff
@@ -0,0 +1,105 @@
+# Static Internationalization [![Build Status](https://travis-ci.org/yola/gulp-static-i18n.svg?branch=master)](https://travis-ci.org/yola/gulp-static-i18n)
+
+> Gulp plugin to translate static assets.
+
+
+## Install
+
+```
+$ npm install --save-dev gulp-static-i18n
+```
+
+
+## Usage
+
+Example gulp file:
+
+```js
+'use strict';
+var gulp = require('gulp');
+var clean = require('gulp-rimraf');
```

Comment:
`gulp-rimraf` is being deprecated in favor of [del](https://github.com/gulpjs/gulp/blob/master/docs/recipes/delete-files-folder.md)